src/pages/CubePlaytestPage.js

```diff
@@ -98,14 +98,23 @@ LabelRow.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
-const CustomDraftCard = ({ format, onEditFormat, onDeleteFormat, ...props }) => {
+const CustomDraftCard = ({
+  format,
+  onEditFormat,
+  onDeleteFormat,
+  onSetDefaultFormat,
+  defaultDraftFormat,
+  ...props
+}) => {
   const { cubeID, canEdit } = useContext(CubeContext);
   const { index } = format;
   return (
     <Card {...props}>
       <CSRFForm method="POST" action={`/cube/startdraft/${cubeID}`}>
         <CardHeader>
-          <CardTitleH5>{format.title} (custom draft)</CardTitleH5>
+          <CardTitleH5>
+            {format.title} (Custom Draft) {defaultDraftFormat === index && '(Default Format)'}
```

Comment:
Having two sets of parens trailing the format titles feels wrong

Reply:
I'm open to suggestions on other ways to demarcate the Default format.

Reply:
How about: Default Format: {name} ({custom draft})